.github/workflows/lint.yaml

```diff
@@ -0,0 +1,20 @@
+name: golangci-lint
+on: [push, pull_request]
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: install go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.16
+      - name: checkout code
+        uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          args: -v --timeout=5m --exclude SA1029
```

Comment:
This lint issue would be a breaking change and should wait until a major version bump.